Content.Server/Pinpointer/NavMapSystem.cs

```diff
@@ -23,19 +28,17 @@ public sealed partial class NavMapSystem : SharedNavMapSystem
 {
     [Dependency] private readonly IAdminLogManager _adminLog = default!;
     [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
-    [Dependency] private readonly SharedMapSystem _mapSystem = default!;
-    [Dependency] private readonly SharedTransformSystem _transformSystem = default!;
+    [Dependency] private readonly TagSystem _tags = default!;
+    [Dependency] private readonly MapSystem _map = default!;
     [Dependency] private readonly IMapManager _mapManager = default!;
-    [Dependency] private readonly IGameTiming _gameTiming = default!;
-    [Dependency] private readonly ITileDefinitionManager _tileDefManager = default!;
+    [Dependency] private readonly TransformSystem _transform = default!;
+
+    private EntityQuery<PhysicsComponent> _physicsQuery;
```

Comment:
your missing a using, Robust.Shared.Physics.Components;

Reply:
No longer relevant

Reply:
"no longer relevant" look at your checks

Reply:
Oh, somehow relevant? I moved back to the commit with EE's stuff